src/domain/Day.ts

```diff
@@ -26,3 +27,9 @@ export function getToday(block: ethereum.Block): Day {
   let date = timestampToDate(block.timestamp)
   return <Day>Day.load(date.toString())
 }
+
+// if the difference between days since nonzerobalance
+// and today's timestamp are greater than or equal to sixty days in seconds
+export function sixtyDays(today: BigInt, nonzerosince: BigInt): boolean {
```

Comment:
Maybe this can also be renamed to a more descriptive name, e.g.:
```suggestion
export function haveSixtyDaysPassed(today: BigInt, nonZeroSince: BigInt): boolean {
```